### PR TITLE
TF Keras layer_utils_test introducing test cases

### PR DESCRIPTION
**Pull request by Dayananda-V**
Test case introducing for layer_utils.py file under keras package.

### CODE REVIEW

**Comment by Dayananda-V** [comment]
@fchollet 

Please help to review the test case

File: tensorflow/python/keras/utils/layer_utils_test.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    def get_model(shape, data_format):
+      model = keras.Sequential()
+      model.add(
+          keras.layers.Conv2D(filters=2, kernel_size=(4, 3),
+                              input_shape=shape, data_format=data_format))
+      model.add(keras.layers.Flatten())
+      model.add(keras.layers.Dense(5))
+      return model
+
+    for data_format in ['channels_first', 'channels_last']:
```
Comment (fchollet) [inline tensorflow/python/keras/utils/layer_utils_test.py]: Use a test parameterization for cases like this, rather than an internal loop in the test. Unit tests should not use loops in general. This applies to a few other of your PRs as well.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: tensorflow/python/keras/utils/layer_utils_test.py (hunk 1 above)
Comment (Dayananda-V) [reply tensorflow/python/keras/utils/layer_utils_test.py]: Sure, will modify according to review comments on this PR and other open PR also. Thanks.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @pavithrasv, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 74 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/keras/utils/layer_utils_test.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```
Comment (pavithrasv) [inline tensorflow/python/keras/utils/layer_utils_test.py]: 2019

**Review by pavithrasv — APPROVED** [review]

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
